src/Guesser/TypeGuesserChain.php

```diff
@@ -60,7 +60,7 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
         }
 
         $bestGuess = TypeGuess::getBestGuess($guesses);
-        // todo - remove `assert` statement after https://github.com/symfony/symfony/pull/37725 is released
+        // todo - remove `assert` statement after https://github.com/phpstan/phpstan-symfony/pull/92 is released
```

Comment:
I think this was not meant to be changed

Reply:
Indeed, I was talking about the link in AclSecurityHandler.php.

Reply:
: facepalm